content/footer.ts

```diff
@@ -0,0 +1,31 @@
+import { NavItem } from './content.types.ts';
+import navItems from './navItems.ts';
+
+export const footerContent = {
```

Comment:
This was a great idea to start a content file for the footer and extract the strings from the footer component file!